docs/userguide/recovery_device.rst

```diff
@@ -0,0 +1,80 @@
+.. Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+.. _doc_userguide_recovery_device:
+
+===============
+Recover devices
+===============
+
+A *recovery device* is a backup of an existing device. If you lose access to Parsec (you forget the password), the recovery device will allow you to recreate a new device and set a new password.
+
+In order to `Create a new recovery device`_ or `Use a recovery device`_ you need to open the recovery device dialog from the main menu:
```

Comment:
Weird capitalization

Reply:
That's to create reference to the titles on the later chapters

Reply:
Then I think the syntax is something like this:
```
`create a new recovery device<Create a new recovery device>`
```

Reply:
TIL: The ref are case insensitive